src/rhoknp/cli/serve.py

```diff
@@ -0,0 +1,150 @@
+import html
+import textwrap
+from enum import Enum
+from typing import Dict, Optional, Union
+
+try:
+    import fastapi
+    import uvicorn
+except ImportError:
+    fastapi = None
+    uvicorn = None
+
+from rhoknp.processors import KNP, KWJA, Jumanpp
+
+
+class AnalyzerType(Enum):
+    """解析器の種類．"""
+
+    jumanpp = "jumanpp"
+    knp = "knp"
+    kwja = "kwja"
```

Comment:
Enum fields are written in UPPER_SNAKE_CASE in rhoknp. Capitalization should be consistent.